### PR TITLE
feat: WEB-759 Update homepage for postgres launch

### PR DESCRIPTION
**Pull request by carlagn**
Fixes #WEB-759

### CODE REVIEW

File: src/data/indexData.ts [hunk 1]
```diff
@@ -1,53 +1,52 @@
...
-      },
-    ],
+  i_studio: {
+    title: "Explore and manipulate your data",
+    eyebrow: "Studio",
+    description: "Interact with your data in a simple tabular interface with full CRUD capabilities, filtering, sorting, and safe editing directly in the model cells.",
+    link: "/studio",
+    icon: "fa-regular fa-table"
+  },
+  t_optimize: {
```
Comment (jharrell) [inline src/data/indexData.ts:24]: what's the distinction of `i` vs `t`?

File: src/pages/index.module.scss [hunk 2]
```diff
@@ -169,84 +213,119 @@ h5.eyebrow {
   line-height: 110%;
   margin: 0;
   text-transform: uppercase;
-  color: var(--secondary-font-color);
+  color: var(--terciary-font-color);
```
Comment (jharrell) [inline src/pages/index.module.scss]: sp: `tertiary`

```suggestion
  color: var(--tertiary-font-color);
```

File: src/data/indexData.ts (hunk 1 above)
Comment (carlagn) [reply src/data/indexData.ts:24]: i = indigo
t = teal

File: src/pages/index.module.scss (hunk 2 above)
Comment (carlagn) [reply src/pages/index.module.scss]: I'm illiterate. thank you

**Review by mhessdev — DISMISSED** [review]

**Review by mhessdev — APPROVED** [review]

**Review by mhessdev — DISMISSED** [review]

**Review by mhessdev — APPROVED** [review]

**Pull request merged**